bin_tests/src/bin/crashtracker_bin_test.rs

```diff
@@ -46,6 +46,7 @@ mod unix {
             Some(ddcommon::Endpoint {
                 url: ddcommon::parse_uri(&output_url)?,
                 api_key: None,
```

Comment:
I guess that all occurrences of these could be removed.

Reply:
you're right. I missed some of them.